modules/persistent-storage-csi-gcp-hyperdisk-storage-pools-procedure.adoc

```diff
@@ -0,0 +1,249 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-gcp-hyperdisk-storage-pools-procedure_{context}"]
+= Setting up hyperdisk-balanced disks
+
+.Prerequisites
+* Access to the cluster with administrative privileges
+
+.Procedure
+To set up hyperdisk-balanced disks:
+
+ifndef::openshift-dedicated[]
+. Create GCP cluster with attached disks provisioned with hyperdisk-balanced disks.
+endif::openshift-dedicated[]
+
+ifndef::openshift-dedicated[]
+. Create a storage class specifying the hyperdisk-balanced disks during installation:
+endif::openshift-dedicated[]
+
+ifndef::openshift-dedicated[]
+.. Follow the procedure in the _Installing a cluster on GCP with customizations_ section.
++ 
+For your install-config.yaml file, use the following example file:
++
+.Example install-config YAML file
+[source, yaml]
+----
+apiVersion: v1
+metadata:
+  name: ci-op-9976b7t2-8aa6b
+
+sshKey: |
+  XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+baseDomain: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+platform:
+  gcp:
+    projectID: XXXXXXXXXXXXXXXXXXXXXX
+    region: us-central1
+controlPlane:
+  architecture: amd64
+  name: master
+  platform:
+    gcp:
+      type: n4-standard-4 <1>
+      osDisk:
+        diskType: hyperdisk-balanced <2>
+        diskSizeGB: 200
+  replicas: 3
+compute:
+- architecture: amd64
+  name: worker
+  replicas: 3
+  platform:
+    gcp:
+      type: n4-standard-4 <1>
+      osDisk:
+        diskType: hyperdisk-balanced <2>
+----
+<1> Specifies the node type as n4-standard-4.
+<2> Specifies the node has the root disk backed by hyperdisk-balanced disk type. All nodes in the cluster should use the same disk type, either hyperdisks-balanced or pd-*.
++
+[NOTE]
+====
+Clusters that use hyperdisk-balanced volumes are not supported, and some nodes do not support it, such as N2 or E2.
```

Comment:
This is a strange statement. I think the whole point was to support _some_ clusters with hyperdisk-balanced volumes.

Reply:
@jsafrane I think that this came from your comment on Jan 23: "Especially, we do not support clusters that use hyperdisk-balanced volumes, but some nodes do not support it, such as n2 or e2."

Should this be: "OCP supports hyperdisk-balanced volumes, but some nodes do not support it, such as N2 and E2."?

Reply:
I think we should say that all nodes in the cluster must support hyperdisk-balanced volumes. We don't support clusters with mixed nodes - say N2 and N3 using hyperdisk-balanced.

Reply:
@jsafrane Fixed. PTAL  and give LGTM if appropriate.

Reply:
lgtm